src/core_codemods/remove_unused_imports.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, codemod_context: CodemodContext, *codemod_args):
         BaseCodemod.__init__(self, *codemod_args)
 
     def transform_module_impl(self, tree: cst.Module) -> cst.Module:
+        # Do nothing in __init__.py files
+        if self.file_context.file_path.name == "__init__.py":
```

Comment:
Is this getting tested by a unit test already?

Reply:
Good catch, I'll add one.